app/mailers/claim_mailer.rb

```diff
@@ -84,6 +84,10 @@ def early_years_payment_provider_email(claim, one_time_password, email)
       magic_link: @magic_link
     }
 
+    if Rails.env.development?
+      Rails.logger.info("\n\nEmail verification code: #{one_time_password}\n")
+    end
+
```

Comment:
Not sure this is related.

Reply:
it's not, no.
I'll put it in a separate commit.